src/kapil/NumbersWhile.java

```diff
@@ -0,0 +1,57 @@
+//WHILE LOOP
+// Print Even numbers.
+// Print  Numbers which is divisible by 5
+// Print Numbers which is divisible by 5 and 3.
+// Print Numbers which is divisible by 7 or 13. 
+package kapil;
+
+public class NumbersWhile {
+
+	void evenNum(int endNumber) {
+		int num =10;
```

Comment:
Why we require num = 10 here?
we need to create a method which accepts two arguments one is start and 2nd is end limit.

Reply:
ok sir